LDAR_Sim/src/initialization/input_manager.py

```diff
@@ -124,31 +123,61 @@ def read_parameter_file(self, filename):
 
         return (new_parameters)
 
+    def handle_parameter_versioning(self, parameters) -> None:
+        if not self.old_params:
+            if 'version' not in parameters:
+                print(('Warning: interpreting parameters as version 3.1 '
```

Comment:
Should be updated to 3.2?

Reply:
Suggestion is to use string formatting and use the CURRENT_MINOR_VERSION and CURRENT_MAJOR_VERSION here instead of hard set text.